platform/android/CHANGELOG.md

```diff
@@ -15,6 +15,33 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 ### Bug fixes
  - Fixed a rendering issue of `collisionBox` when `text-translate` or `icon-translate` is enabled. [#15467](https://github.com/mapbox/mapbox-gl-native/pull/15467)
 
+## 8.4.0-alpha.1 - September 4, 2019
+[Changes](https://github.com/mapbox/mapbox-gl-native/compare/android-v8.3.0...android-v8.4.0-alpha.1) since [Mapbox Maps SDK for Android v8.3.0](https://github.com/mapbox/mapbox-gl-native/releases/tag/android-v8.3.0):
+
+### Features
+- Bindings for aggregated cluster properties [#15425](https://github.com/mapbox/mapbox-gl-native/pull/15425)
+- Add font array attribute configuration [#15488](https://github.com/mapbox/mapbox-gl-native/pull/15488) 
+- Expose camera padding option [#15444](https://github.com/mapbox/mapbox-gl-native/pull/15444) 
+- Add number format expression [#15424](https://github.com/mapbox/mapbox-gl-native/pull/15424) 
+
+### Bug fixes
+- Initialize CameraPositionActivity listeners after the style is loaded [#15484](https://github.com/mapbox/mapbox-gl-native/pull/15484)
+- Cancel CameraAnimatorActivity joint example when the map is destroyed [#15500](https://github.com/mapbox/mapbox-gl-native/pull/15500)
+- Invalidate foreground icon sources even when location layer is hidden [#15507](https://github.com/mapbox/mapbox-gl-native/pull/15507)
+- Fixed unnecessary layers blink when the same components are re-added after the style reload [#15513](https://github.com/mapbox/mapbox-gl-native/pull/15513)
+- Fixed an issue in JNI marshaling code that converted a float to long that produced invalid expressions [#15557](https://github.com/mapbox/mapbox-gl-native/pull/15557)
+- Call callback onFinish() immediately when trying to move camera to current position [#15558](https://github.com/mapbox/mapbox-gl-native/pull/15558)
+
+### Build
+- Reorder android-check make command [#15493](https://github.com/mapbox/mapbox-gl-native/pull/15493)
```

Comment:
this doesn't really impact normal usage, can be removed

Reply:
Removed